stories/example-screen-staff-list.stories.js

```diff
@@ -289,322 +289,105 @@ const tabsDoubleRowHTML = `
       </a>
     </li>
   </ul>
-  <ul class="nav nav-tabs bg-white border border-light border-top-0 ps-7 pe-7">
-    <li class="nav-item">
-      <a class="nav-link active fs-5" href="#">
+
+  <ul class="nav nav-tabs bg-white ps-7 pe-7">
+    <li class="nav-item py-2 px-3">
+      <a class="fs-6 text-primary" href="#" style="font-weight: 400;">
         <span>All Jobs</span>
       </a>
     </li>
-    <li class="nav-item">
-      <a class="nav-link fs-5" aria-current="page" href="#">
+    <li class="nav-item p-2">
+      <a class="fs-6 text-muted" href="#" style="font-weight: 400;">
         <span>Recommended Jobs</span>
       </a>
     </li>
-    <li class="nav-item">
-      <a class="nav-link fs-5" href="#">
+    <li class="nav-item p-2">
+      <a class="fs-6 text-muted" href="#" style="font-weight: 400;">
         <span>My Favorites</span>
       </a>
     </li>
-    <li class="nav-item">
-      <a class="nav-link fs-5" href="#">
+    <li class="nav-item p-2">
+      <a class="fs-6 text-muted" href="#" style="font-weight: 400;">
         <span>Applications</span>
       </a>
     </li>
   </ul>
 `;
 
+const batchAndPagination = `
+  <div class="d-flex align-items-center justify-content-between flex-wrap mx-3 mt-4 mb-3">
+    <div class="d-flex align-items-center justify-content-center">
+      <select class="form-select fs-6" aria-label="Default select example" style="padding-right: 2rem; line-height: 1.3;">
```

Comment:
Inline styles for padding and line-height need to be replaced with named classes here.

Padding classes are created here: https://github.com/Symplicity/bootstrap/blob/v5.0.2-janus/scss/_variables.scss

And use the spacers here: https://github.com/Symplicity/bootstrap/blob/v5.0.2-janus/scss/_variables.scss#L258

I use this calculator to do the conversion: https://www.ninjaunits.com/converters/pixels/rem-pixels/

So in this case you'd want to replace `padding-right: 20px;` with `pe-7` (since 1.25rem = 20px in the calculator).

For the `line-height: 1.3` you can try using one of the 4 pre-defined line-height helper classes here: https://getbootstrap.com/docs/5.0/utilities/text/#line-height (use whatever is close, we can create a custom style for this later if we need to).